docs/content/documentation/templates/taxonomies.md

```diff
@@ -3,11 +3,15 @@ title = "Taxonomies"
 weight = 40
 +++
 
-Zola will look up the following files in the `templates` directory:
+Zola will look up the following, taxon-specific files in the `templates` directory:
 
 - `$TAXONOMY_NAME/single.html`
 - `$TAXONOMY_NAME/list.html`
 
+if they are not found, it will attempt to fall back on the following generic template files:
+- `single.html`
+- `list.html`
```

Comment:
can we call them `taxonomy_single.html` and `taxonomy_list.html`? So you know exactly what they are from the name

Reply:
Absolutely, just pushed a commit now.